### DISCUSSION TITLE
Runtime on largest datasets?

### DISCUSSION THREAD

**Discussion by samihaija** [snap-stanford/ogb#271] [General]
Any chance we can find out the run-time on largest datasets of baselines? E.g., for full-batch GCN and GraphSage on ogbl-citation2, say, if I get a GPU with large enough memory to execute the runs, would it take hours or days to finish the training pipeline?

**Comment by weihua916** [comment]
If I remember correctly, each run of ogbl-citation2 takes about a few hours, so 10 runs will take about a day.
